packages/clerk-js/src/ui/styles/_userprofile.scss

```diff
@@ -67,7 +67,7 @@ $content-width-lg: 620px;
       border-right-color: transparent;
       width: 26px;
       height: 26px;
-      border-radius: (26px / 2);
+      border-radius: 50%;
```

Comment:
This was throwing a warning for not using `calc`. Based on context, seems like border radius `50%` will do the same thing for cheaper